contracts/test/TestWarmColdAccount.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.12;
+/* solhint-disable no-inline-assembly */
+
+import "../interfaces/IAggregatedAccount.sol";
+import "../interfaces/IEntryPoint.sol";
+
+
+contract TestWarmColdAccount is IAggregatedAccount {
+    IEntryPoint private ep;
+    uint public state = 1;
+    constructor(IEntryPoint _ep) payable {
+        ep = _ep;
```

Comment:
what the ep is for (I also wanted to say it sholud be immutable, so it won't be in cold/warm storage)